services/ui-src/src/components/reports/sections/StaticDrawerSection.tsx

```diff
@@ -51,13 +57,13 @@ export const StaticDrawerSection = ({ form, page, setLoading }: Props) => {
     }
   };
 
-  const entityRows = (entities: string[]) =>
+  const entityRows = (entities: EntityShape[]) =>
     entities.map((entity) => (
-      <Flex key={entity} sx={sx.entityRow}>
-        <Heading as="h5">{entity}</Heading>
+      <Flex key={entity.id} sx={sx.entityRow}>
+        <Heading as="h5">{entity.name}</Heading>
         <Button
           sx={sx.enterButton}
-          onClick={() => openRowDrawer(entity)}
+          onClick={() => openRowDrawer(entity.name)}
```

Comment:
Question: I would think we'd want to pass through the entity id and name here so we can tell it which one to update later in the drawer. Right?